FordLabs/PeopleMover
# Adding accessibility linting to Peoplemover

## Issue
Connects #388 

## What was done
- [x] Added jsx-a11y for accessibility linting
- [x] Added react-axe for testing application live on chrome, reports findings in chrome dev tools

## How to test
1. See that jsx-a11y is linting our code
2. Run app locally and check chrome dev tools for console log of findings
